src/main/java/uk/gov/ons/census/casesvc/utility/FieldworkHelper.java

```diff
@@ -1,21 +1,66 @@
 package uk.gov.ons.census.casesvc.utility;
 
+import org.springframework.util.StringUtils;
 import uk.gov.ons.census.casesvc.model.entity.Case;
 
 public class FieldworkHelper {
-  public static boolean shouldSendCaseToField(Case caze, String eventChannel) {
-    if (eventChannel.equals("FIELD")) {
```

Comment:
removal of the "don't send back to FIELD" has been lost from multiple places - see other comments

Reply:
have discussed - this requirement seems to have gone away